qemu-cc/debian.rules

```diff
@@ -0,0 +1,66 @@
+#!/usr/bin/make -f
+include /usr/share/dpkg/default.mk
+export LANG=C
+%:
+	dh $@
+
+override_dh_auto_configure:
+	dh_auto_configure -- \
+	--prefix=/usr \
+	--datadir=/usr/share/qemu-cc \
+	--libdir=/usr/lib64/qemu-cc \
+	--libexecdir=/usr/libexec/qemu-cc \
+	--disable-tools \
```

Comment:
All these options should really be coming from https://github.com/clearcontainers/packaging/blob/master/scripts/configure-hypervisor.sh.